cluster/common.sh

```diff
@@ -2,15 +2,6 @@
 
 set -e
 
-function install_cnao {
```

Comment:
Why we do not need this anymore?

Reply:
It's part of the `kubevirtci` repo [now](https://github.com/kubevirt/kubevirtci/tree/master/cluster-up/cluster/k8s-1.17#bringing-the-cluster-up-with-cluster-network-addons-operator-provisioned). It's enabled just by passing `KUBEVIRT_WITH_CNAO=true`

Reply:
To make it effect, this must be merged before: https://github.com/kubevirt/project-infra/pull/449

Reply:
OK, thanks